src/mumble/Log.cpp

```diff
@@ -415,20 +415,26 @@ QString Log::validHtml(const QString &html, bool allowReplacement, QTextCursor *
 
 	qtd.adjustSize();
 	QSizeF s = qtd.size();
-
-	if (!valid || (s.width() > qr.width()) || (s.height() > qr.height())) {
-		qtd.setPlainText(html);
-		qtd.adjustSize();
-		s = qtd.size();
-
-		if ((s.width() > qr.width()) || (s.height() > qr.height())) {
-			QString errorMessage = tr("[[ Text object too large to display ]]");
-			if (tc) {
-				tc->insertText(errorMessage);
-				return QString();
-			} else {
-				return errorMessage;
-			}
+	int messageSize = s.width() * s.height();
+	int allowedSize = 300 * 11288;
```

Comment:
Should include a comment about this being a "max area", so as long as the bounding box of the rendered HTML stays within this area, it will be allowed.

Also: why 300 * 11288 instead of something like 2048x2048?